src/desktop/components/split_test/running_tests.coffee

```diff
@@ -15,10 +15,18 @@
 # module.exports = {}
 
 module.exports = {
- onboarding_test:
-   key: 'onboarding_test'
-   outcomes:
-     control: 50
-     experiment: 50
-   edge: 'experiment'
+  onboarding_test:
+    key: 'onboarding_test'
+    outcomes:
+      control: 50
+      experiment: 50
+    edge: 'experiment'
+  artist_page:
+    key: 'artist_page'
+    outcomes:
+      control: 25
+      no_info: 25
+      no_header: 25
+      no_info_header: 25
+    edge: 'experiment'
```

Comment:
The `edge` is what admins get sent to, so it should be one of the 4 test options.